push-down-test/src/main.go

```diff
@@ -243,15 +272,17 @@ func diffRunResult(
 }
 
 func buildDefaultConnStr(port int) string {
-	return fmt.Sprintf("root@tcp(localhost:%d)/?allowNativePasswords=true", port)
+	return fmt.Sprintf("root@tcp(localhost:%d)/{db}?allowNativePasswords=true", port)
```

Comment:
Why make this change.

Reply:
Oh, got it. but I think replace is not a good idea. Why not use a place holder?

Reply:
It is a placeholder (by using replace to fill the place after that) 😅

Reply:
Do you have some suggestion to do it more elegantly?